app/src/main/java/org/oppia/android/app/devoptions/vieweventlogs/EventLogItemViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.android.app.devoptions.vieweventlogs
+
+import org.oppia.android.app.model.EventLog
+import org.oppia.android.app.viewmodel.ObservableViewModel
+import org.oppia.android.util.system.OppiaDateTimeFormatter
+import javax.inject.Inject
+
+/** [ViewModel] for displaying a event log. */
+class EventLogItemViewModel @Inject constructor(
+  val eventLog: EventLog,
+  private val oppiaDateTimeFormatter: OppiaDateTimeFormatter
+) : ObservableViewModel() {
+
+  fun processDateAndTime(): String {
+    return oppiaDateTimeFormatter.formatDateFromDateString(
+      OppiaDateTimeFormatter.DD_MMM_hh_mm_aa,
+      eventLog.timestamp
+    )
+  }
+
+  fun processPriority(): String? {
```

Comment:
'processPriority' isn't really clear--what does 'process' mean here/what's actually happening? Ditto for the other pubic API functions here.

Reply:
Renamed function to `formatPriorityString`. SImilarly renamed other functions as well. Also, added KDoc to explain what it does. PTAL. Thanks.